src/app/search/search-workflow-table/search-workflow-table.component.ts

```diff
@@ -35,10 +35,15 @@ import { SearchService } from '../state/search.service';
 export class SearchWorkflowTableComponent extends SearchEntryTable implements OnInit {
   readonly entryType = 'workflow';
   public dataSource: MatTableDataSource<Workflow>;
-  public galaxyShortfriendlyName = DescriptorLanguageService.workflowDescriptorTypeEnumToShortFriendlyName(
-    Workflow.DescriptorTypeEnum.Gxformat2
-  ).split(' ')[0];
-  constructor(dateService: DateService, searchQuery: SearchQuery, searchService: SearchService) {
+  public galaxyShortfriendlyName = this.descriptorLanguageService
```

Comment:
Added comment and removed the '.split(' ')[0];' which it seems is not necessary, since the friendly name is just 'Galaxy'. Also added a test to check this.